### PR TITLE
tutorial: Stargate Voter structural and user exp edits

### PR DESCRIPTION
**Pull request by barriebyron**
Now that the Stargate poll tutorial is updated and published, I am running the tutorial and making content changes for structure, style, and how it works.

### CODE REVIEW

File: voter/index.md [hunk 1]
```diff
@@ -55,372 +101,433 @@ Cosmos SDK's version is: Stargate v0.40.0 (or above)
...
+```
+
+1. In the `<script>` tag, import the component like this:
 
-```js
+```javascript
 import PollForm from "../components/PollForm";
 import PollList from "../components/PollList";
 
 export default {
```
Comment (barriebyron) [inline voter/index.md:520]: @toschdev do we leave the `export` section here?

File: voter/index.md [hunk 2]
```diff
@@ -720,58 +836,56 @@ export default {
...
+The `App.vue` file handles the transactions of the components. Modify the `<script>` tag to look like:
 
-```js
+```javascript
+<script>
 import './scss/app.scss'
 import '@starport/vue/lib/starport-vue.css'
 import Sidebar from './components/Sidebar'
 
 export default {
```
Comment (barriebyron) [inline voter/index.md:851]: hmmm I didn't make changes here, is it my editor settings? important?

File: voter/index.md [hunk 3]
```diff
@@ -809,53 +925,52 @@ For the votes you can go to the API page on [http://localhost:1317/username/vote
 }
 ```
 
-The endpoint paths are defined by the username you use when bootstrapping the application with Starport, together with your module name.
+The endpoint paths are defined by the username you used when bootstrapping the application with Starport, together with the module name.
```
Comment (barriebyron) [inline voter/index.md:928]: for example?

File: voter/index.md [hunk 4]
```diff
@@ -864,47 +979,45 @@ import sdk "github.com/cosmos/cosmos-sdk/types"
...
 
 ```go
 app.voterKeeper = *voterkeeper.NewKeeper(
   appCodec, keys[votertypes.StoreKey], keys[votertypes.MemStoreKey], app.BankKeeper,
 )
 ```
 
-Now you are ready to use all the bank functions that you added to the expected keepers file above.
-Next you will define how the transaction will require the funds in order to exectue the transaction.
+<!-- here is how far I got, but my app doesn't work yet --> Now, you are ready to use all the bank functions that you added to the expected keepers file above. Next, you need to define how the transaction will require the funds in order to exectue the transaction.
```
Comment (barriebyron) [inline voter/index.md:1020]: @toschdev I really really wanted to succeed in this long-winded, very detailed tutorial... I got farther than I did before, but will benefit from some technical 1:1 

welcome back!

**Comment by the-frey** [comment]
Cool, have forked & added the typo fixes and whatnot here https://github.com/cosmos/sdk-tutorials/pull/654 to bring it in line with what I did to complete the tutorial 👍 
Hopefully some of that is useful!

**Comment by barriebyron** [comment]
@the-frey your input is sound and appreciated! I've added the updates from https://github.com/cosmos/sdk-tutorials/pull/654 to this working PR. Take a look, or wait until we publish this iteration and see updates then. You are the best!

**Comment by the-frey** [comment]
Thanks, LGTM

**Review by toschdev — APPROVED** [review]

**Pull request merged**